.github/workflows/check-label-in-progress.yml

```diff
@@ -0,0 +1,17 @@
+name: Check label in-progress to prevent merging
```

Comment:
Do not hesitate to comment your workflow

Reply:
You are right. Comment will be added.